artemis-jms-client/src/test/java/org/apache/activemq/artemis/jms/client/ConversionTest.java

```diff
@@ -36,7 +36,7 @@ public void testCoreToJMSConversion() {
       clientMessage.setDurable(true)
               .setPriority((byte) 9)
               .setExpiration(123456);
-      Map<String, Object> messageMap = clientMessage.toMap();
+      Map<String, Object> messageMap = clientMessage.toMap(0);
```

Comment:
See other comment but semantically i would 0 to truncate to 0, unlimited i would expect to call -1, likewise typically i would use toMap() to keep interface unchanged.

Reply:
agree